pkg/entity/batch_job/cmd.go

```diff
@@ -0,0 +1,77 @@
+package batch_job
+
+import (
+	"github.com/spf13/cobra"
+	"strmprivacy/strm/pkg/common"
+)
+
+const (
+	batch_jobs_configuration_flag_name = "file"
```

Comment:
As said before, maybe rename this to batch_job_file_flag_name?

Reply:
I renamed it to this to `batch_jobs_configuration_flag_name` ;) But your proposal is also good.